### PR TITLE
refs #000: enforce ipv6=off when the enforcing is enabled

### PR DESCRIPTION
**Pull request by Monska85**

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -74,6 +74,7 @@ export NGINX_OSB_PUBLIC_PATH="${NGINX_OSB_PUBLIC_PATH:-}"
 export NGINX_OSB_ASSETS_PATH="${NGINX_OSB_ASSETS_PATH:-}"
 export NGINX_ASSETS_STREAM_OVER_S3="${NGINX_ASSETS_STREAM_OVER_S3:-0}"
 export NGINX_OSB_RESOLVER="${NGINX_OSB_RESOLVER:-8.8.8.8 ipv6=off}"
+export NGINX_OSB_RESOLVER_ENFORCE_IPV6_OFF="${NGINX_OSB_RESOLVER_ENFORCE_IPV6_OFF:-1}"
```
Comment (grota) [inline docker-entrypoint.sh:77]: is it BC to default to 1 here?

Comment (Monska85) [reply docker-entrypoint.sh:77]: No, at the moment we want to enforce all our installations to avoid the ipv6 to resolve the DNS for the OSB proxy pass.

Comment (grota) [reply docker-entrypoint.sh:77]: ok

**Review by grota — APPROVED** [review]

**Review by paolomainardi — APPROVED** [review]
LGTM

**Pull request merged**